### PR TITLE
feat(SearchBox-Headless,react & vue)/add enableTelemetry def

### PR DESCRIPTION
**Pull request by mohdashraf010897**
**PR Type**: Feature

**Description**: Adding telemetry support , adds an `enableTelemetry` property in `appbaseConfig` object to switch off telemetry.

 ### Affected Libs: 
- SearchBase
- React-SearchBox
- Vue-SearchBox
- React Native SearchBox

[🔗  Implementation PR](https://github.com/appbaseio/searchbox/pull/116)

### CODE REVIEW

File: content/docs/reactivesearch/react-native-searchbox/searchbase.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ nestedSidebar: 'react-native-searchbox-reactivesearch'
     -   **enableQueryRules** `boolean` If `false`, then appbase.io will not apply the query rules on the search requests. Defaults to `true`.
     -   **userId** `string` It allows you to define the user id to be used to record the appbase.io analytics. Defaults to the client's IP address.
     -   **customEvents** `Object` It allows you to set the custom events which can be used to build your own analytics on top of appbase.io analytics. Further, these events can be used to filter the analytics stats from the appbase.io dashboard.
+     -   **enableTelemetry** `Boolean` When set to `false`, disable the telemetry. Defaults to `true`.
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/react-native-searchbox/searchbase.md]: @mohdashraf010897 There looks like an indent issue with the PR here, can you fix this?

**Review by siddharthlatest — CHANGES_REQUESTED** [review]
Follow the formatting fixes that we've made in #220.

File: content/docs/reactivesearch/react-native-searchbox/searchbase.md (hunk 1 above)
Comment (mohdashraf010897) [reply content/docs/reactivesearch/react-native-searchbox/searchbase.md]: https://github.com/appbaseio/Docs/pull/221/commits/97f5892385ed283fa7f9dad84b1584cefed51eae

Comment (mohdashraf010897) [reply content/docs/reactivesearch/react-native-searchbox/searchbase.md]: fixed here @siddharthlatest  https://github.com/appbaseio/Docs/commit/97f5892385ed283fa7f9dad84b1584cefed51eae

**Review by siddharthlatest — APPROVED** [review]

**Pull request merged**